kafka_utils/util/zookeeper.py

```diff
@@ -101,8 +101,13 @@ def get_brokers(self, names_only=False):
 
         :rtype : dict of brokers
         """
-        broker_ids = self.get_children("/brokers/ids")
-
+        try:
+            broker_ids = self.get_children("/brokers/ids")
+        except NoNodeError:
+            _log.error(
```

Comment:
Let' print a debug or info message here rather than error. And the same for get_topics()